presto-parser/src/main/java/com/facebook/presto/sql/tree/Select.java

```diff
@@ -64,7 +64,7 @@ public boolean isDistinct()
     @Override
     public List<Node> getChildren()
     {
-        return null;
+        return ImmutableList.copyOf(selectItems);
```

Comment:
No need for a copy here, since it's already an immutable list. (the copy is a no-op, so technically it's fine, but we don't do this on other parts of the code so it's confusing)